### PR TITLE
[Merged by Bors] - feat: port Analysis.Calculus.LHopital

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

- [x] depends on: #4249

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#4249
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#4249~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/Analysis/Calculus/LHopital.lean [hunk 1]
```diff
@@ -195,26 +181,24 @@ theorem lhopital_zero_atTop_on_Ioi (hff' : ∀ x ∈ Ioi a, HasDerivAt f (f' x)
 theorem lhopital_zero_atBot_on_Iio (hff' : ∀ x ∈ Iio a, HasDerivAt f (f' x) x)
     (hgg' : ∀ x ∈ Iio a, HasDerivAt g (g' x) x) (hg' : ∀ x ∈ Iio a, g' x ≠ 0)
     (hfbot : Tendsto f atBot (𝓝 0)) (hgbot : Tendsto g atBot (𝓝 0))
-    (hdiv : Tendsto (fun x => f' x / g' x) atBot l) : Tendsto (fun x => f x / g x) atBot l :=
-  by
+    (hdiv : Tendsto (fun x => f' x / g' x) atBot l) : Tendsto (fun x => f x / g x) atBot l := by
   -- Here, we essentially compose by `has_neg.neg`. The following is mostly technical details.
```
Comment (semorrison) [inline Mathlib/Analysis/Calculus/LHopital.lean:185]:
```suggestion
  -- Here, we essentially compose by `Neg.neg`. The following is mostly technical details.
```

**Comment by semorrison** [comment]
bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Pull request closed without merging**